lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
The problem is that if history.utime == 0, the if statement will not be executed because history.utime == 0 is falsey.  And history.utime is always 0 for a kernel module so it always goes in the else block for kernel modules which causes incorrect values.  The proposed change is to specifically check for undefined (instead of any falsey value) so that the if block will be entered with utime=0 and the else block will only be executed if utime is undefined which was the intention I think.